build/Jamfile.v2

```diff
@@ -140,13 +140,12 @@ lib boost_stacktrace_from_exception
   : # requirements
     <warnings>all
     <target-os>linux:<library>dl
-    <target-os>windows:<build>no  # not supported at the moment
 
     # Command line option to disable build
     <boost.stacktrace.from_exception>off:<build>no
 
     # Require usable libbacktrace on other platforms
-    [ check-target-builds ../build//libbacktrace : : <build>no ]
+    #[ check-target-builds ../build//libbacktrace : : <build>no ]
```

Comment:
I have no idea how to enable this conditionally.

Reply:
Me too :)

Seems fine for now. If something goes wrong I'll fix it by myself (probably with some help from the b2 developers)